deploy-help/delete-database.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash -e
+
+cluster_name="$1"
+deployment_dir=~/deployment
+kubernetes_build_dir="$deployment_dir/concent-deployment/kubernetes/build"
+
+function run_kubernetes_script {
+    executed_script="$1"
+    cd $kubernetes_build_dir/
+    ./$executed_script
+}
+
+if [ -z "$cluster_name" ]; then
+    echo 'Please specify the name of database'
+    exit 1
+fi
+
+# Confirm executing script
+read -p "Warning must execute after deploy.sh script. Are you sure about execute this script? It's destroy database[y/n]" -n 1 -r
+echo
+if [[ $REPLY =~ ^[Yy]$ ]]; then
+
+    playbooks=(
+        job-cleanup.yml
+        reset-db.yml
+    )
+
+    run_kubernetes_script delete-services.sh
+    (
```

Comment:
You should not need really need to call the deployment scripts directly here. Playbooks should be able to do this for you. I have created an issue to split `deploy.yml` into two playbooks: #281. When it's done, update this script to use it.